browser/src/map/handler/Map.Keyboard.js

```diff
@@ -863,6 +863,12 @@ L.Map.Keyboard = L.Handler.extend({
 		case this.keyCodes.PERIOD: // .
 			app.socket.sendMessage('uno .uno:SuperScript');
 			return true;
+		case this.keyCodes.SPACE:
```

Comment:
We have new array for adding shortcuts in more unified and verified way.
https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.KeyboardShortcuts.ts#L161

Could you try to add that one there?

Reply:
Thanks @eszkadev for the suggestion! I've updated the implementation and it works as expected.